apps/site/src/pages/settings/billing-settings-panel/payment-method.tsx

```diff
@@ -35,75 +35,68 @@ export const PaymentMethod: FunctionComponent<{
   }, [paymentMethod]);
 
   return (
-    <Box display="flex" alignItems="center" marginTop={2} marginBottom={2}>
-      <CardLogoIcon
-        cardBrand={paymentMethod.card?.brand}
-        sx={{ fontSize: 40, color: ({ palette }) => palette.gray[40] }}
-      />
-      <Box marginLeft={2}>
-        <Typography
-          variant="bpMicroCopy"
-          component="p"
-          sx={{ marginBottom: 1 }}
-        >
-          <strong>
-            {paymentMethod.card?.brand
-              ? cardBrandToHumanReadable(paymentMethod.card?.brand)
-              : ""}
-          </strong>
-          <Fade in={isDefault}>
-            <Box
-              component="span"
-              sx={({ palette, spacing }) => ({
-                padding: spacing(0.5, 0.75),
-                color: palette.purple[700],
-                background: palette.purple[20],
-                borderRadius: 1,
-                textTransform: "uppercase",
-                fontSize: 11,
-                marginLeft: 1,
-              })}
-            >
-              <FontAwesomeIcon
-                icon={faCheck}
-                sx={{ marginRight: 0.75, fontSize: 12 }}
-              />
-              <strong>Default</strong>
-            </Box>
-          </Fade>
-          {isExpired ? (
-            <Box
-              component="span"
-              sx={({ palette, spacing }) => ({
-                padding: spacing(0.5, 0.75),
-                color: palette.gray[70],
-                background: palette.gray[10],
-                borderRadius: 1,
-                textTransform: "uppercase",
-                fontSize: 11,
-                marginLeft: 1,
-              })}
-            >
-              <CircleExclamationRegularIcon
-                sx={{ marginRight: 0.75, fontSize: 12 }}
-              />
-              <strong>Expired</strong>
-            </Box>
-          ) : null}
-        </Typography>
-        <Box display="flex">
-          <Typography variant="bpMicroCopy" component="p">
-            Ending <strong>*{paymentMethod.card?.last4}</strong>
-          </Typography>
+    <Box
+      display="flex"
+      alignItems="center"
+      justifyContent="space-between"
+      sx={sx}
+    >
+      <Box display="flex" alignItems="center">
+        <CardLogoIcon
+          cardBrand={paymentMethod.card?.brand}
+          className="card-logo-icon"
+          sx={{ fontSize: 40, color: ({ palette }) => palette.gray[40] }}
+        />
+        <Box marginLeft={2}>
           <Typography
             variant="bpMicroCopy"
             component="p"
-            sx={{ marginLeft: 1.5 }}
+            className="card-brand"
```

Comment:
Same question for this one

Reply:
I'm also using this to change the color of the text: https://github.com/blockprotocol/blockprotocol/blob/8dc5c992ae5b78b6d98a2d900200afaaf1a8de5e/apps/site/src/pages/settings/billing/upgrade/change-payment-method-modal.tsx#L153-L155